contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -61,7 +61,7 @@ contract BosonVoucherBase is IBosonVoucher, BeaconClientBase, OwnableUpgradeable
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[43] private __gap;
+    uint256[42] private __gap;
```

Comment:
This should stay 43, since we don't have any new storage variable anymore.

Reply:
forgot to put it back, done